src/test/java/uk/gov/hmcts/reform/judicialbooking/util/ValidationUtilTest.java

```diff
@@ -19,6 +19,11 @@
 
 class ValidationUtilTest {
 
+    static String uuidString1 = "5629957f-4dcd-40b8-a0b2-e64ff5898b28";
+    static String uuidString2 = "5629957f-4dcd-40b8-a0b2-e64ff5898b29";
```

Comment:
the string `uuidString2` is not used anywhere

Reply:
Addressed